app/controllers/submissions/new/basics.js

```diff
@@ -65,7 +65,7 @@ export default class SubmissionsNewBasics extends Controller {
 
   @action
   async loadTab(gotoRoute) {
-    if (!this.doiInfo.title) set(this, 'doiInfo.title', get(this, 'publication.title'));
```

Comment:
This is the actual change that fixed things for the bug.

Reply:
Note, stashing this information on a piece of Doi Info is something maybe we should consider changing earlier. It took me a while to even hunt this down in part because of the naming.

Also note, you can proceed initially with a DOI and then backtrack and remove it and use a manually inputted title and journal and this still all works.